SmartTutor/Repository/ActiveEducationRepository/ActivityRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using SmartTutor.ActiveEducationModel;
+using SmartTutor.Repository.ActiveEducationRepository;
+using System.Collections.Generic;
+
+namespace SmartTutor.Repository
+{
+    public class ActivityRepository : IActiveEducationRepository
```

Comment:
ActivityInMemoryRepository would be more suitable, although I would change the "Activity" part based on the previous comments.

Activity is a really generic word that should be avoided (especially as a primary identifier of a concept, like its used here).